### ISSUE TITLE
aliases show up in each api id

### ISSUE THREAD

**Issue by justin2004** [basilapi/basil#129]
i deployed 3 different APIs and gave them each a distinct alias but when i run:
```sh
curl -X GET -u user:pass 'http://localhost:8080/basil/'
```

i get:
```json
[               
  {             
    "id": "vbq9jcoixqht",
    "modified": 1698099217408,
    "name": "null",         
    "alias": [                                                         
      "query-2",
      "query-1",
      "query-3"      
    ],                        
    "createdBy": "penelope",
    "location": "http://localhost:8080/basil/vbq9jcoixqht"
  },
  {
    "id": "ure2aab9fyas",
    "modified": 1698099217606,
    "name": "null",
    "alias": [
      "query-2",
      "query-1",
      "query-3"
    ],
    "createdBy": "penelope",
    "location": "http://localhost:8080/basil/ure2aab9fyas"
  },
  {
    "id": "t4nwop7s4z7b",
    "modified": 1698099217861,
    "name": "null",
    "alias": [
      "query-2",
      "query-1",
      "query-3"
    ],
    "createdBy": "penelope",
    "location": "http://localhost:8080/basil/t4nwop7s4z7b"
  }
]
```

shouldn't each API id only have 1 alias?

**Comment by enridaga** [comment]
The Horror!

**Comment by enridaga** [comment]
This should be fixed now in [9839549](https://github.com/basilapi/basil/commit/9839549126b52ea78a61561936508d2d712a7501)

**Comment by justin2004** [comment]
it looks good now. thanks!
